lib/trento/domain/host/commands/rollup_host.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Trento.Domain.Commands.RollUpHost do
+  @moduledoc """
+  Start an host aggregate rollup
```

Comment:
Spelling: Not entirely sure but this should be `Start a host` as the `h` is pronounced.
The same would apply in the events docs